seed/index.js

```diff
@@ -0,0 +1,55 @@
+// const House = require('../db/Schemas/Listing.js');
+
+
+//loop 100 times
+const seed = () => {
+    const houses = [];
+    for (let i = 0; i < 100; i++) {
+        const house = {};
+        let pad = i;
+
+        if (pad.toString().length < 2) {
+            house['Listing_id'] = '00' + pad.toString();
+        } else {
+            house['Listing_id'] = '0' + pad.toString();
+        }
+
+        house['Price'] = Math.floor(Math.random() * 5999999) + 100000;
+
+        const Detail = {};
+
+        Detail['Room_count'] = Math.floor(Math.random() * 6) + 2;
+        Detail['Bathroom_count'] = Math.floor(Math.random() * 3) + 1;
+        Detail['Square_footage'] = Math.floor(Math.random() * 2600) + 500;;
+
+        house['Details'] = Detail;
+
+        const Address = {};
+        Address['House_number'] = Math.floor(Math.random() * 9888) + 111;
+        Address['Street_number'] = Math.floor(Math.random() * 300);
+
+        const street = ['street', 'avenue', 'place', 'boulevard'];
+        const direction = ['', ' NW', ' NE', ' S', ' SW', ' SE', ' N'];
+
+        Address['Street'] = street[Math.floor(Math.random() * 4)] + direction[Math.floor(Math.random() * 7)];
+        Address['City'] = 'Seattle';
+        Address['State'] = 'WA';
+        Address['ZIP_code'] = Math.floor(Math.random() * 99) + 98100;
+        house['Address'] = Address;
+
+
+        const bools = [true, false];
+
+        house['Zestimate'] = bools[Math.floor(Math.random() * 2)];
+        house['Agent'] = bools[Math.floor(Math.random() * 2)];
```

Comment:
I like your way of doing this. 
Did you consider adding any controls so that a closer to real number of houses were listed with agents? I ask because apparently "91% of sellers were assisted by a real estate agent when selling their home" as of 2018; src= https://www.nar.realtor/research-and-statistics/quick-real-estate-statistics

Reply:
All dummy data, but that is helpful.  I'll look at factoring that into my option